docs/guides/profile/cpu.md

```diff
@@ -21,21 +21,149 @@ kernel stack traces.
+Capturing stack traces...
+Capturing stack traces... Hit Ctrl-C to end.^C
+NODE             NAMESPACE        POD              CONTAINER        COMM             PID    COUNT
+minikube         default          random           random           cat              17680   1
+        entry_SYSCALL_64_after_hwframe
+        do_syscall_64
+        vfs_read
+        ksys_read
+        urandom_read_nowarn.isra.0
+        _extract_crng
+...
+minikube         default          random           random           cat              17680   1
+        entry_SYSCALL_64_after_hwframe
+        do_syscall_64
+        vfs_read
+        ksys_read
+        urandom_read_nowarn.isra.0
+        _extract_crng
+```
+
+This gadget also supports custom column outputting, for example:
+
+```bash
+$ kubectl gadget profile cpu --timeout 1 --podname random -o custom-columns=node,pod
+Capturing stack traces...
+NODE             POD
+minikube         random
+...
+minikube         random
+```
+
+The following command is the same as default printing:
+
+```bash
+$ kubectl gadget profile cpu --timeout 1 --podname random -o custom-columns=node,namespace,pod,container,comm,pid,count,stack
+Capturing stack traces...
+NODE             NAMESPACE        POD              CONTAINER        COMM             PID    COUNT
+minikube         default          random           random           cat              17680  1
+        entry_SYSCALL_64_after_hwframe
+        do_syscall_64
+        vfs_read
+        ksys_read
+        urandom_read_nowarn.isra.0
+        __lock_text_start
+        [unknown]
+        [unknown]
+...
+minikube         default          random           random           cat              17680  1
+        entry_SYSCALL_64_after_hwframe
+        do_syscall_64
+        vfs_read
+        ksys_read
+        urandom_read_nowarn.isra.0
+        __lock_text_start
+        [unknown]
+        [unknown]
+```
+
+You can also use JSON as output:
+
+```bash
+$ kubectl gadget profile cpu --timeout 1 --podname random -o json | jq
+{
+  "node": "minikube",
+  "namespace": "default",
+  "pod": "random",
+  "container": "random",
+  "comm": "cat",
+  "pid": 65969,
+  "user_stack": [
+    "[unknown]",
+    "[unknown]"
+  ],
+  "kernel_stack": [
+    "__lock_text_start",
+    "__lock_text_start",
+    "urandom_read_nowarn.isra.0",
+    "vfs_read",
+    "ksys_read",
+    "do_syscall_64",
+    "entry_SYSCALL_64_after_hwframe"
+  ],
+  "value": 1
+}
+...
+{
+  "node": "minikube",
+  "namespace": "default",
+  "pod": "random",
+  "container": "random",
+  "comm": "cat",
+  "pid": 65969,
+  "user_stack": [
+    "[unknown]",
+    "[unknown]"
+  ],
+  "kernel_stack": [
+    "__lock_text_start",
+    "__lock_text_start",
+    "_crng_backtrack_protect",
+    "urandom_read_nowarn.isra.0",
+    "vfs_read",
+    "ksys_read",
+    "do_syscall_64",
+    "entry_SYSCALL_64_after_hwframe"
+  ],
+  "value": 1
+}
+```
+
+Note that piping with `jq` will only works when using `--timeout`.
```

Comment:
Do you know why this happens? Other gadgets like `./kubectl-gadget trace exec -o json -A | jq` or `./kubectl-gadget top file -o json -A | jq` work fine.